src/helpers/prepare-queued-pr-for-merge.ts

```diff
@@ -46,9 +46,9 @@ export const updatePrWithMainline = async (pullRequest: PullRequest | SimplePull
   } catch (error) {
     if ((error as GithubError).status === 204) {
       core.info('The first PR in the queue is already up to date!');
-    }
-    if ((error as GithubError).status === 409) {
-      core.info('The first PR in the queue has a merge conflict.');
+    } else if ((error as GithubError).status >= 400) {
```

Comment:
Won't we want to log out an error with status < 400?

Reply:
anything in the 200s is still a success message so i don't really think it's worth failing the job over. i guess i'm assuming that 100s/300s won't ever happen with the way we're using octokit for all of our API requests

Reply:
I'm talking about logging with `core.info`. As it stands we wouldn't do anything with status < 400 right? Since if that if statement condition is not met it just skips that whole block?

Reply:
ah yeah i can just log it otherwise, or just always log it and set failure based on these conditions